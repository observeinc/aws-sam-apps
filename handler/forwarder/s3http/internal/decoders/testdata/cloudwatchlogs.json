{"messageType":"DATA_MESSAGE","owner":"123456789012","logGroup":"example-log-group","logStream":"example-log-stream","subscriptionFilters":["example-filter"],"logEvents":[{"id":"1","timestamp":1620678200,"message":"Error: Unable to connect to database server"},{"id":"2","timestamp":1620678300,"message":"Warning: High CPU usage detected"},{"id":"3","timestamp":1620678400,"message":"Info: Application started successfully"},{"id":"4","timestamp":1620678500,"message":"Error: Disk space running low"},{"id":"5","timestamp":1620678600,"message":"Info: User 'admin' logged in"}]}
{"messageType":"DATA_MESSAGE","owner":"987654321098","logGroup":"another-log-group","logStream":"another-log-stream","subscriptionFilters":["filter-1","filter-2"],"logEvents":[{"id":"1","timestamp":1620679200,"message":"Info: Server restarted"},{"id":"2","timestamp":1620679300,"message":"Error: Service crashed unexpectedly"},{"id":"3","timestamp":1620679400,"message":"Warning: Memory usage exceeding 90%"},{"id":"4","timestamp":1620679500,"message":"Info: New user registered"},{"id":"5","timestamp":1620679600,"message":"Error: Network connection lost"}]}
