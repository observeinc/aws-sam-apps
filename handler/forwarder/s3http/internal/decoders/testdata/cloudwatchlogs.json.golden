{"id":"1","timestamp":1620678200,"message":"Error: Unable to connect to database server","owner":"123456789012","logGroup":"example-log-group","logStream":"example-log-stream","subscriptionFilters":["example-filter"],"messageType":"DATA_MESSAGE"}
{"id":"2","timestamp":1620678300,"message":"Warning: High CPU usage detected","owner":"123456789012","logGroup":"example-log-group","logStream":"example-log-stream","subscriptionFilters":["example-filter"],"messageType":"DATA_MESSAGE"}
{"id":"3","timestamp":1620678400,"message":"Info: Application started successfully","owner":"123456789012","logGroup":"example-log-group","logStream":"example-log-stream","subscriptionFilters":["example-filter"],"messageType":"DATA_MESSAGE"}
{"id":"4","timestamp":1620678500,"message":"Error: Disk space running low","owner":"123456789012","logGroup":"example-log-group","logStream":"example-log-stream","subscriptionFilters":["example-filter"],"messageType":"DATA_MESSAGE"}
{"id":"5","timestamp":1620678600,"message":"Info: User 'admin' logged in","owner":"123456789012","logGroup":"example-log-group","logStream":"example-log-stream","subscriptionFilters":["example-filter"],"messageType":"DATA_MESSAGE"}
{"id":"1","timestamp":1620679200,"message":"Info: Server restarted","owner":"987654321098","logGroup":"another-log-group","logStream":"another-log-stream","subscriptionFilters":["filter-1","filter-2"],"messageType":"DATA_MESSAGE"}
{"id":"2","timestamp":1620679300,"message":"Error: Service crashed unexpectedly","owner":"987654321098","logGroup":"another-log-group","logStream":"another-log-stream","subscriptionFilters":["filter-1","filter-2"],"messageType":"DATA_MESSAGE"}
{"id":"3","timestamp":1620679400,"message":"Warning: Memory usage exceeding 90%","owner":"987654321098","logGroup":"another-log-group","logStream":"another-log-stream","subscriptionFilters":["filter-1","filter-2"],"messageType":"DATA_MESSAGE"}
{"id":"4","timestamp":1620679500,"message":"Info: New user registered","owner":"987654321098","logGroup":"another-log-group","logStream":"another-log-stream","subscriptionFilters":["filter-1","filter-2"],"messageType":"DATA_MESSAGE"}
{"id":"5","timestamp":1620679600,"message":"Error: Network connection lost","owner":"987654321098","logGroup":"another-log-group","logStream":"another-log-stream","subscriptionFilters":["filter-1","filter-2"],"messageType":"DATA_MESSAGE"}
